src/components/SquareBox/SquareBox.tsx

```diff
@@ -33,8 +35,18 @@ const styles = StyleSheet.create({
 
 export function SquareBox() {
     const id = React.useId();
-        const [squares, setSquares] = React.useState<Array<string>>([]);
-      
+        const [squares, setSquares] = React.useState<Array<Object>>([]);
+const addCircle=function(){
+   
+    setSquares(squares => ([
+        ...squares,
+        {color: randomElement(Object.values(COLORS)),type:'circle'}]
+        
+        
```

Comment:
Looks like you have some problem with the code formatting - please check why do you have here a two new lines